File: jetpack_smart_display/app/src/main/java/com/smsoft/smartdisplay/data/BluetoothDeviceType.kt
package com.smsoft.smartdisplay.data

enum class BluetoothDeviceType(val id: String, val title: String) {
    THERMOBEACON("thermobeacon", "ThermoBeacon"),
    ATC("atc", "ATC_");

    companion object {
        fun toList(): List<String> {
            val list = mutableListOf<String>()
            entries.forEach {
                list.add(it.title)
            }
            return list
        }
    }
}

File: flutter_smart_display/lib/models/bluetooth_device_type.dart
enum BluetoothDeviceType {
  thermobeacon("ThermoBeacon"),
  atc("ATC_");

  const BluetoothDeviceType(this.title);
  final String title;

  static List<String> toList() {
    List<String> result = [];
    for (var element in BluetoothDeviceType.values) {
      result.add(element.title);
    }
    return result;
  }
}